### DISCUSSION TITLE
解决了续签ssl问题

### DISCUSSION THREAD

**Discussion by ws20940548** [chaitin/SafeLine#794] [General]
经研究，实现了续签ssl，无论泛域名、单域名、多域名同证

1. 可以直接拉取宝塔更新后的证书更新
2. acme的证书
3. 其他例如certbot的证书(我还是喜欢certbot，泛域名出来的证书目录不带*，哈哈)

逻辑理顺了，代码还没写。
不知道搭建这么个服务大家有没有需求
不是小白的话完全可以自己搞
搭建的话，爱发电或者几块钱一个月？
但是等搭建完，官方升级了这个的话，那也是白忙活^_^

**Comment by Lvshujun0918** [comment]
讲讲思路？

**Reply by ws20940548** [reply]
https://www.iots.vip/post/update-safeline-ssl-cert-from-file.html
